airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -110,8 +110,8 @@
   // defaults
   private static final String DEFAULT_SPEC_CACHE_BUCKET = "io-airbyte-cloud-spec-cache";
   public static final String DEFAULT_JOB_KUBE_NAMESPACE = "default";
-  private static final String DEFAULT_JOB_CPU_REQUIREMENT = null;
-  private static final String DEFAULT_JOB_MEMORY_REQUIREMENT = null;
+  private static final String DEFAULT_JOB_CPU_REQUIREMENT = "0.2";
+  private static final String DEFAULT_JOB_MEMORY_REQUIREMENT = "500Mi";
```

Comment:
I added this because I noticed that these were set being set to `null` if the `JOB_MAIN_CONTAINER_...*` environment variables are not set to anything in the `.env` files, and I believe we always want these to at least be set to these defaults if these environment variables are not set. But please correct me if I'm wrong!